pages/1.12/cli/autocompletion/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle:  CLI Autocompletion
+title: CLI Autocompletion
+menuWeight: 7
+excerpt: Enabling autocompletion for the CLI
+enterprise: false
+---
+
+Autocompletion was realeased with the 1.12 release so the CLI must be of version 0.7.x or greater. \
```

Comment:
s/realeased/released

Reply:
I'm not sure that users are aware of the CLI version they are using, asking them to make sure they have downloaded the CLI using the link available on the UI seems safer that telling `the CLI must be of version 0.7.x or greater`.